src/components/controls/OpacityControls.js

```diff
@@ -111,11 +111,12 @@ export default class OpacityControls extends Component {
   }
 
   componentWillReceiveProps(nextProps) {
+    const range = nextProps.state.model.range;
```

Comment:
```suggestion
    const { min, max, units} = nextProps.state.model.range;
```
then use those below

Reply:
Done (: